homeassistant/components/here_travel_time/coordinator.py

```diff
@@ -81,20 +91,36 @@ async def _async_update_data(self) -> HERETravelTimeData | None:
             departure,
         )
 
-        response = await self._api.route(
-            transport_mode=TransportMode(self.config.travel_mode),
-            origin=here_routing.Place(origin[0], origin[1]),
-            destination=here_routing.Place(destination[0], destination[1]),
-            routing_mode=route_mode,
-            arrival_time=arrival,
-            departure_time=departure,
-            return_values=[Return.POLYINE, Return.SUMMARY],
-            spans=[Spans.NAMES],
-        )
-
-        _LOGGER.debug("Raw response is: %s", response)
+        try:
+            response = await self._api.route(
+                transport_mode=TransportMode(self.config.travel_mode),
+                origin=here_routing.Place(origin[0], origin[1]),
+                destination=here_routing.Place(destination[0], destination[1]),
+                routing_mode=route_mode,
+                arrival_time=arrival,
+                departure_time=departure,
+                return_values=[Return.POLYINE, Return.SUMMARY],
+                spans=[Spans.NAMES],
+            )
+            _LOGGER.debug("Raw response is: %s", response)
```

Comment:
Move the rest of this out of the try/catch to the reduce its scope as much as possible, so its clear the interval reset and parsing logic won't raise this exception. (Here and below)

Reply:
Thanks, done!